frontend/src/views/Raid.vue

```diff
@@ -173,10 +173,16 @@
                       </div>
                   </div>
                   <div class="col-lg-12 join-raid">
-                    <button class="claim-btn" @click="promptRewardClaim()" v-tooltip="'Rewards from Previous Raid'">
+                    <button class="claim-btn" :class="rewardIndexes !== null && rewardIndexes.length > 0 ? 'opacity-1': 'opacity-0'"
+                      @click="promptRewardClaim()" v-tooltip="rewardIndexes !== null && rewardIndexes.length > 0 ?
+                      $t('raid.rewardsFromPrevious'): $t('raid.noAvailableRewards')">
```

Comment:
This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled=""  there should be a few examples around, you shouldn't need the css then either

Reply:
copy

> This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled="" there should be a few examples around, you shouldn't need the css then either

noted

Reply:
> This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled="" there should be a few examples around, you shouldn't need the css then either

I already tried to add method that trigger the disabled but the opacity is not affected..